principles.md

```diff
@@ -0,0 +1,25 @@
+## Product and Engineering Principles
+
+**Simpler always wins in the end**
+
+We choose simpler designs whenever possible. Having less configuration knobs
+is better for security. Complex products are easier to hack and harder
+to use. Security feature is only good, when it's easy to use.
+
+That's why we stick to [client certificates](https://goteleport.com/teleport/how-it-works/certificate-based-authentication-ssh-kubernetes/)
```

Comment:
this isn't actually true is it? IE we don't use client certs for AAP, and gravity was a long term user for tokens for long term access, I don't know if teleport is similar or not.

Reply:
I can add whenever possible to clarify. Even with AAP the dream is to switch to mTLS, we were constrained by web browsers not supporting client certs in our design.